change/@fluentui-react-focus-2020-09-02-15-55-10-list-finddomnode.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "none",
+  "comment": "Fix typo in docs  ",
```

Comment:
```suggestion
  "comment": "Fix typo in docs",
```

Reply:
I did that so that beachball change wouldn't force me to use the commit message from a previous PR 😬  (though obviously it can be manually edited later)